ktor-server/ktor-server-core/jvmAndNix/src/io/ktor/server/response/ApplicationResponse.kt

```diff
@@ -26,6 +26,11 @@ public interface ApplicationResponse {
      */
     public val headers: ResponseHeaders
 
+    /**
+     * Indicates that this response is already committed and no further changes are allowed.
+     */
+    public val isSent: Boolean
```

Comment:
maybe `isCommited`?

Reply:
yep, it's event sounds better in terms of documentation